public/javascripts/other.js

```diff
@@ -58,7 +58,19 @@ function geoloc()
         geocoder.geocode( { 'address': lat + ', ' + lng }, (results, status) => {
             if (status == 'OK') 
             {
-                document.getElementById('location').value = results[0].address_components[2].long_name + ', ' + results[0].address_components[4].long_name + ', Australia';
+                let locality, state, country;
+                const addressComponents = results[0].address_components;
+
+                for (i = 0; i < addressComponents.length; ++i)
```

Comment:
This code can be cleaned up with es6 new syntax
```
for (const [k, v] of addressComponents) {}
```
I feel the if and else if statements could also be cleaned up